playbooks/robusta_playbooks/pod_investigator_enricher.py

```diff
@@ -75,6 +77,13 @@ def detect_pod_issue(pod: Pod) -> PodIssue:
     return PodIssue.NoneDetected
 
 
+def get_pod_issue_details(pod: Pod) -> str:
+    # Works/should work only or KubeContainerWaiting and KubePodNotReady
+    # Note: in line with the old code in pod_issue_investigator, we only get the message for
+    # the first of possibly many misbehaving containers.
+    return pod.status.containerStatuses[0].state.waiting.message
```

Comment:
This might fail, if there are no `containerStatuses` or `waiting` is None
Please verify this doesn't fail on the edge cases

Reply:
fixed